firebase-perf/dev-app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,9 @@
         <meta-data
             android:name="fragment_sampling_percentage"
             android:value="100.0" />
+        <meta-data
+            android:name="experiment_as_ttid"
```

Comment:
I'd prefer experiment_app_start_ttid so that it is more obvious that it is for app start.

Reply:
done